src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+  action="/search"
+   method="get"
+   data-qa="big"
+   class="form"
+   >
```

Comment:
Fix formatting in whole html document:
```suggestion
    <form
       action="/search"
       method="get"
       data-qa="big"
       class="form"
   >
```